resources/compass/charts/director/templates/dashboard-configmap.yaml

```diff
@@ -0,0 +1,1709 @@
+{{- if and .Values.global.metrics.enabled -}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "fullname" . }}-dashboard
+  namespace: kyma-system
+  labels:
+    grafana_dashboard: "1"
+    app: monitoring-grafana
+data:
+  compass-director-dashboard.json: |-
```

Comment:
What about moving the JSON content to separate file and using Helm's `.Files.Get` function?

Reply:
I will do it in next iteration, okay? Because I will modify the dashboard anyway.